libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -887,32 +879,6 @@ - (void) updateCredentials:(NSDictionary *) params {
 
 }
 
-- (void)configureWebUserAgent
```

Comment:
These methods don't actually set or reset the user agent on the web view. They simply change the user defaults.
It's enough when using cordova (because it looks for the user agent in the user defaults and sets it on its web view) but it doesn't do anything otherwise.